spec/abci/abci++_methods.md

```diff
@@ -835,13 +835,16 @@ Most of the data structures used in ABCI are shared [common data structures](../
     | Name              | Type                    | Description                                                                  | Field Number |
     |-------------------|-------------------------|------------------------------------------------------------------------------|--------------|
     | validator         | [Validator](#validator) | The validator that sent the vote.                                            | 1            |
-    | signed_last_block | bool                    | Indicates whether or not the validator signed the last block.                | 2            |
-    | vote_extension    | bytes                   | Non-deterministic extension provided by the sending validator's Application. | 3            |
+    | vote_extension    | bytes                   | Non-deterministic extension provided by the sending validator's Application. | 2            |
+    | extension_signature | bytes             | Vote extension signature created by CometBFT.                | 3            |
```

Comment:
```suggestion
    | extension_signature | bytes             | Signature of the vote extension produced by the sending validator and verified by CometBFT.                | 4            |
```

Reply:
This was copied from the code comment, the rest is in line with that .

Reply:
What code? We should then update the code comment, probably, ehehe.